### DISCUSSION TITLE
exporting the GrandMeister panel from Mac to Linux ?

### DISCUSSION THREAD

**Discussion by moooooooo** [RomanKubiak/ctrlr#548] [Q&A]
is there anyway i can export the panel that used the the GrandMeister Deluxe Ctrlr app for MacOS so that i can use its panel in Ctrlr for Linux ?
If so let me know how. Advanced commands are welcome. Experienced sysadmin here :-)
cheers
Peter

**Comment by damiensellier** [comment]
Hi,
It was not available as a .panel?
Can you please provide the link?
Thanks

**Reply by moooooooo** [reply]
sorry for the delay. Been extreely ill.
ok, this the Windows and macOS d/l URL.
I have the macOS version and would love if i could somehow export the panel so i can use it on Limux.
cheers
https://ctrlr.org/hughes-kettner-remote/

**Reply by damiensellier** [reply]
Hi, hope you're doing better mate! In the macOS app package there's a folder called "Resources" which contains  the panel specific files addons to ctrlr. I don't know how the structure of a linux app is done but if it's the same as mac, you need to copy / paste those resources to a linux stock app duplicate named after the panel, and it will work maybe?  
Maybe it would require modding the info.plist as well to match the panel details in the different xml keys.
Not sure about that though, I've never don that before. You never heard back from the panel dev' ?

<img width="476" alt="macos app structure" src="https://github.com/RomanKubiak/ctrlr/assets/55164385/7b521fad-993c-4144-9178-8cb2dacb2842">

**Reply by moooooooo** [reply]
thx yeah i'm getting there. i thought i'd gone into the Resources folder and checked that. 
i did install a 3rd party panel on the Linux version, so i'll see what that file structure looks like on macOS with this panel and try and mimic that on Linux. 
My mac is intel so the files *may* work on Linux if they are normal files but i'll post back when i can.

Thx for the info - it'll be good if i can get this running Linux.
Cheers

**Reply by damiensellier** [reply]
Pretty sure you can get it working from there with a little bit of work. Good luck!
